lib/demo/state_decorators/pairing_data_provider.dart

```diff
@@ -41,5 +41,6 @@ class DemoPairingData extends PairingData {
           "DemoPairingId",
           Uint8List(0),
           sodium.crypto.box.keyPair(),
+          null,
```

Comment:
Should this be "DemoPairingRemark" also?

Reply:
We can have it, but we are not using this value right now, for successful pairing the pairing remark is null.